### ISSUE TITLE
feat (or bug?): Allow `Enter` key event on a masked input

### ISSUE THREAD

**Issue by stephannv** [codermarcos/simple-mask-money#67]
Pressing enter on an masked input isn't submitting the form.

I'm using 4.1.0 version.

To reproduce save this as a html and open it on a browser. Try to press enter on masked input and then try to press enter on not masked input:
```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
  <script src="https://github.com/codermarcos/simple-mask-money/releases/download/v4.1.0/simple-mask-money.umd.js"></script>

</head>
<body>
  <form action="https://www.google.com">
    <label for="my-input">Masked input (enter doesn't work)</label>
    <input type="text" id="my-input">

    <br>
    <br>

    <label for="other-input">Not masked input (enter works)</label>
    <input type="text" id="other-input" placeholder="press enter here">

    <input type="submit" value="Submit">
  </form>

  <script>
    SimpleMaskMoney.setMask('#my-input');
  </script>
</body>
</html>

```

**Comment by codermarcos** [comment]
Thank you very much for your issue we need more developers as you ❤️
It should be fixed at the version 4.1.2. Could you check please?

**Comment by stephannv** [comment]
Thanks, I appreciate your effort with this project.

I already upgrade simple-mask-money to 4.1.2 on my project, and this issue is fixed now. Thanks again.
